tests/expectations/test_expectation.py

```diff
@@ -225,6 +226,40 @@ def test_expectation_configuration_property():
     )
 
 
+@pytest.mark.unit
+def test_expectation_configuration_window():
+    expectation = gxe.ExpectColumnMaxToBeBetween(
+        column="foo",
+        min_value=0,
+        max_value=10,
+        windows=[
```

Comment:
With respect to your optional comment above, you can have tests where you don't set `windows` and one where you set it to `None` and see serialization works correctly.

Reply:
o nice, yeah

Reply:
Strangely, `None` succeeds an input output, but `[]` on input is being ignored on output. Looking into it

Reply:
update: It was the `dict(exclude_defaults=True)`, updated default and getting expected results